HW_1/task4.py

```diff
@@ -0,0 +1,10 @@
+s = int(input('Введите общее количество бумажных журавликов: '))
+petya = s/6
+serezha = s/6
+katya = 2*s/3
```

Comment:
Неплохо бы добавить проверку на кратность 6. Если у вас количество журавликов не будет кратно 6, то результат будет получаться дробным, а такого быть не может. Мы же не можем сделать пол журавлика. Ну или на крайний случай можно использовать целочисленное отделение. Кроме того, не принято использовать в качестве имен переменных функций и так далее транслитерацию русских слов.
```suggestion
petya = serezha = s // 6
katya = 2 * s // 3
```